spec/contracts.dd

```diff
@@ -122,8 +124,18 @@ body
     ...
 }
 ------
-	In an out statement, $(I result) is initialized and set to the
-	return value of the function.
+	$(P When the return type is not `void`, and no identifier is
+	declared, a variable named `__result` is defined by the compiler
+	for the `out` contract and initialized to the return value of the
+	function:)
```

Comment:
OK, maybe `__result` shouldn't be documented then. @zachthemystic quoted use of it to me [here](https://github.com/dlang/DIPs/pull/79#discussion_r126827866), I assumed it was a language feature.